sdk/dev-env/windows/manifests/nodejs-18.json

```diff
@@ -0,0 +1,49 @@
+{
+    "homepage": "https://nodejs.org",
+    "description": "As an asynchronous event driven JavaScript runtime, Node is designed to build scalable network applications. (Long Term Support)",
+    "version": "18.20.8",
```

Comment:
I notice we're using 18.12 on Linux and 18.20 here - can we bump to 18.20 on Linux as well? I don't think we should waste any time on this in this PR unless it's trivial to do - I'd rather we merge this with the discrepancy.